migrate/repo_checks.py

```diff
@@ -494,7 +494,14 @@ def _update_branch_protection(self, params):
         )
         resp = requests.put(url, headers=headers, json=params)
 
-        resp.raise_for_status()
+        try:
+            resp.raise_for_status()
+        except requests.HTTPError as e:
+            # If the branch is missing, the repo is probably new and has no branches
+            # ignore that for now.
+            if resp.json.get("message") != "Branch not found":
+            ):
```

Comment:
This is a syntax error. I can open a PR to fix it, but I'm wondering: when and where is this script run? If it did run, did it trigger an error?

Reply:
@regisb thanks for the catch, the script is still being run manually but in the future will hopefully run as a github action periodically.  I can take care of fixing before we run it again.

Reply:
https://github.com/openedx/terraform-github/pull/55